src/components/accordion/accordion.yaml

```diff
@@ -34,6 +36,7 @@ examples:
 
 - name: with all sections already open
```

Comment:
is there a need for this? Unless we support 'maintain state' aren't accordions always going to start closed?

Reply:
@joelanman not always, it seems. One service got in touch with me to say that they have different sections open by default depending upon answers the user has previously given.

Reply:
Since we are going to support maintaining state, I think this is okay. Resolving but feel free to unresolve if needed.